packages/contracts/src/contracts/current/protocol/TokenTransferProxy/TokenTransferProxy.sol

```diff
@@ -16,71 +16,23 @@
 
 */
 
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.21;
```

Comment:
This file should not be changed - we want to test against the exact same bytecode as the existing deployed version.